app/code/Magento/Bundle/Model/Product/SelectionProductsDisabledRequired.php

```diff
@@ -128,7 +129,8 @@ public function getChildProductIds(int $bundleId, ?int $websiteId = null): array
     private function getProducts(array $selectionProductIds, int $websiteId): array
     {
         $productIds = [];
-        $defaultStoreId = $this->storeManager->getWebsite($websiteId)->getDefaultStore()->getId();
+        $defaultStore = $this->storeManager->getWebsite($websiteId)->getDefaultStore();
+        $defaultStoreId = $defaultStore ? $defaultStore->getId() : Store::DEFAULT_STORE_ID;
         foreach ($selectionProductIds as $optionProductIds) {
             $productIds = array_merge($productIds, $optionProductIds);
         }
```

Comment:
Please fix array_merge in a loop:
```suggestion
        foreach ($selectionProductIds as $optionProductIds) {
            $productIds[] = $optionProductIds;
        }
        $productIds = array_merge([], ...$productIds); 
```

Reply:
Hello @ihor-sviziev, thanks for your review, I fixed code by you recommendation.

Thank you